doc/source/guide/acquiring_data/fido.rst

```diff
@@ -0,0 +1,68 @@
+-----------------------------
+Downloading Data using Fido
+-----------------------------
+
+This guide outlines how to search for and download data using SunPy's
+Federated Internet Data Obtainer...or more usually (and
+sanely) referred to as Fido.  Fido is a unified API for seeking
```

Comment:
Yeah we decided to make it `Fido` in the code because the backronymn was a bit of a joke really. I suggest we keep referring to it as `Fido` for consistency